### ISSUE TITLE
wandb can't find validation accuracy

### ISSUE THREAD

**Issue by besbesmany** [ultralytics/yolov5#4048]
## ❔Question
I run yolov5 , I confused with many charts 
I don't know where is the val accuracy chart
also what are the meaing of "cls_loss" , "box_loss",  "obj_loss" 
also what are "mAp_:0.5" and  "mAp_:0.5:0.95"

which two of them are the training/validation accuracy ?
and 
which two of them are the training/validation loss?

## Additional context

![1](https://user-images.githubusercontent.com/35992924/126068952-e1205aa8-f20d-470d-b3fe-1865bec59415.png)
![2](https://user-images.githubusercontent.com/35992924/126068959-9830a7ac-6398-4728-8a13-fea0c76e0f6b.png)

**Comment by glenn-jocher** [comment]
@besbesmany you might want to start with some basics: https://en.wikipedia.org/wiki/Precision_and_recall

**Comment by besbesmany** [comment]
Thank you I will read them carefully 
but can you tell me 
which two of them are the training/validation accuracy ?
and
which two of them are the training/validation loss?
